services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -597,21 +577,31 @@ public Builder language(@Nullable Locale language) {
     /**
      * Whether or not to return additional metadata along the route. Possible values are:
      * {@link DirectionsCriteria#ANNOTATION_DISTANCE},
-     * {@link DirectionsCriteria#ANNOTATION_DURATION},
      * {@link DirectionsCriteria#ANNOTATION_DURATION} and
-     * {@link DirectionsCriteria#ANNOTATION_CONGESTION}. Several annotation can be used by
-     * separating them with {@code ,}.
+     * {@link DirectionsCriteria#ANNOTATION_CONGESTION}
+     *
+     * @param annotations a list of annotations
+     * @return this builder for chaining options together
+     */
+    public Builder annotations(@NonNull List<String> annotations) {
+      this.annotations = annotations;
+      return this;
+    }
+
+    /**
+     * Whether or not to return additional metadata along the route. Possible values are:
+     * {@link DirectionsCriteria#ANNOTATION_DISTANCE},
+     * {@link DirectionsCriteria#ANNOTATION_DURATION} and
+     * {@link DirectionsCriteria#ANNOTATION_CONGESTION}
      *
-     * @param annotations string referencing one of the annotation direction criteria's. The strings
-     *                    restricted to one or multiple values inside the {@link AnnotationCriteria}
-     *                    or null which will result in no annotations being used
+     * @param annotation string referencing one of the annotation direction criteria's.
      * @return this builder for chaining options together
      * @see <a href="https://www.mapbox.com/api-documentation/navigation/#route-leg-object">RouteLeg object
      * documentation</a>
      * @since 2.1.0
      */
-    public Builder annotations(@Nullable @AnnotationCriteria String... annotations) {
-      this.annotations = annotations;
+    public Builder addAnnotation(@AnnotationCriteria @NonNull String annotation) {
```

Comment:
This is breaking SemVer. What about deprecating `annotations` and add `addAnnotation`?

Reply:
fixed